src/Maintenance/Maintenance/help/Get-AzApplyUpdate.md

```diff
@@ -0,0 +1,176 @@
+Module Name: Az.Maintenance
+online version: https://docs.microsoft.com/en-us/powershell/module/az.maintenance/get-azapplyupdate
+schema: 2.0.0
+---
+
+# Get-AzApplyUpdate
+
+## SYNOPSIS
+Track maintenance updates to resource
+
+## SYNTAX
+
+```
+Get-AzApplyUpdate [-ResourceGroupName] <String> [[-ResourceParentType] <String>]
+ [[-ResourceParentName] <String>] [-ProviderName] <String> [-ResourceType] <String> [-ResourceName] <String>
+ [-ApplyUpdateName] <String> [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Track maintenance updates to resource
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-AzApplyUpdate -ResourceGroupName smdtest$region -ResourceParentType hostGroups -ResourceParentName smddhg$region -ResourceType hosts -ResourceName smddh$region -ProviderName Microsoft.Compute -ApplyUpdateName default
+
+
+Status         : InProgress
+ResourceId     : /subscriptions/42c974dd-2c03-4f1b-96ad-b07f050aaa74/resourcegroups/smdtestwestus2/providers/Microsoft.Compute/hostGroups/smddhgwestus2/hosts/smddhwestus2
+LastUpdateTime : 11/8/2019 9:39:01 AM
+Id             : /subscriptions/42c974dd-2c03-4f1b-96ad-b07f050aaa74/resourcegroups/smdtestwestus2/providers/Microsoft.Compute/hostGroups/smddhgwestus2/hosts/smddhwestus2/providers/Microsoft.Maintenance/applyUpdates/default
+Name           : default
+Type           : Microsoft.Maintenance/applyUpdates
+```
+
+Track maintenance updates to resource
+
+## PARAMETERS
+
+### -ApplyUpdateName
+applyUpdate Id
+
+```yaml
+Type: System.String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: 7
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.Core.IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzContext, AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ProviderName
+Resource provider name
+
+```yaml
+Type: System.String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: 4
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ResourceGroupName
+Resource group name
+
+```yaml
+Type: System.String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: 1
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ResourceName
+Resource identifier
+
+```yaml
+Type: System.String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: 6
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ResourceParentName
+{{ Fill ResourceParentName Description }}
```

Comment:
Still plenty parameters without help messages. That's why you get these `{{ Fill ... }}`

Reply:
these are all resolved